x-pack/plugins/apm/public/components/alerting/helper.ts

```diff
@@ -24,3 +24,14 @@ export function getAbsoluteTimeRange(windowSize: number, windowUnit: string) {
     end: now,
   };
 }
+
+export function isNewApmRuleFromStackManagement(
+  alertParams: any,
+  metadata?: AlertMetadata
+) {
+  return (
+    alertParams !== undefined &&
+    Object.keys(alertParams).length === 0 &&
+    metadata === undefined
+  );
+}
```

Comment:
```suggestion
  return (
    alertParams &&
    !Object.keys(alertParams).length &&
    !metadata
  );
}
```

Or did you want to make it explicit?

Reply:
I did want to make it explicit. `alertParams` must be `{}` and `metadata` must be `undefined`.